test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/ArrayModelBinderTest.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.AspNetCore.Http.Internal;
 using Moq;
 using Xunit;
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
This wasn't needed before and the `async Task` methods all contain `await` operators. Is it still needed and, if so, where does the problem lie?